### ISSUE TITLE
Adding more genres

### ISSUE THREAD

**Issue by terrysahaidak** [andrewngu/sound-redux#26]
What about adding more genres? It could be like first i choose something like rock, metal, trance etc, and then indie-rock, alternative, pop-rock etc.

**Comment by andrewngu** [comment]
Good idea! I'm coming up with a way you can select your genres, maybe even custom ones...

**Comment by mansour-ahmed** [comment]
Hello @andrewngu , I came across this app and i kind of like it! would like to help out with the implementation of new genres! have you already been working on something? where can I start from?
